rust/src/shards/gui/misc/mod.rs

```diff
@@ -0,0 +1,30 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright © 2022 Fragcolor Pte. Ltd. */
+
+use crate::core::registerShard;
+use crate::types::ExposedTypes;
+use crate::types::ParamVar;
+
+struct GetClipboard {
+  parents: ParamVar,
+  requiring: ExposedTypes,
+}
+
+struct SetClipboard {
+  parents: ParamVar,
+  requiring: ExposedTypes,
+}
+
+struct Reset {
+  parents: ParamVar,
+  requiring: ExposedTypes,
+}
+
+mod clipboard;
+mod reset;
+
+pub fn registerShards() {

```

Comment:
Wouldn't it be better if we just added clipboard shards that interact with the system clipboard (probably though SDL_Get/SetClipboardText or rust crate) that way they can also be used outside of a UI context and would work the same way

Reply:
Probably better yes.